frontend/src/graphql/userPoints.graphql.types.ts

```diff
@@ -27,9 +28,8 @@ export type UserPointsQuery = {
       userByTeacherId: {
         __typename?: "Users";
         userId: string;
-        secondName: string;
+        fullName?: string | null;
```

Comment:
is `null` necessary here?
```suggestion
        fullName?: string;
```

Reply:
It's auto generated, so I wouldn't change that